src/_components/ForNPOs/ProjectCriteria.jsx

```diff
@@ -0,0 +1,126 @@
+  return (
+    <figure className="flex-col flex w-full lg:p-5 lg:max-w-[1280px]">
+      <div className="m-24">
+        <h1 className="text-4xl font-semibold pb-6">
+          {" "}
+          What We're Looking For{" "}
+        </h1>
+        <div className="flex">
+          <div className="flex flex-col">
+            {/* Left Side Content */}
+            <div className="flex">
+              <div>
+                {" "}
+                <img
+                  className="size-1"
+                  src="../../assets/vector/higlobealt.svg"
+                  alt="Globe"
+                />{" "}
+              </div>
+              <div className="font-semibold text-3xl text-primary">
+                {" "}
+                <p> {"Impact"} </p>{" "}
+              </div>
+            </div>
+            <div>
+              {" "}
+              <p className="font-normal text-xl pb-2">
+                {" "}
+                {
+                  "What is the purpose and goal of this project? Our team prioritizes projects that generate social impact and contribute to the greater community."
+                }{" "}
+              </p>{" "}
+            </div>
+            <div className="flex">
+              <div>
+                {" "}
+                <img
+                  className="size-1"
+                  src="../../assets/vector/mdoutlinetimeline.svg"
+                  alt="Clock"
+                />{" "}
+              </div>
+              <div className="font-semibold text-3xl text-primary">
+                {" "}
+                <p> {"Scope"} </p>{" "}
+              </div>
+            </div>
+            <div>
+              {" "}
+              <p className="font-normal text-xl pb-2">
+                {" "}
+                {
+                  "Is this project feasible within our timeline? We want to adhere to your deadlines while delivering high-quality results in a reasonable time frame."
+                }{" "}
+              </p>{" "}
+            </div>
+            <div className="flex">
+              <div>
+                {" "}
+                <img
+                  className="size-1"
+                  src="../../assets/vector/hilightbulb.svg"
+                  alt="Clock"
+                />{" "}
+              </div>
+              <div className="font-semibold text-3xl text-primary">
+                {" "}
+                <p> {"Challenge"} </p>{" "}
+              </div>
+            </div>
+            <div>
+              {" "}
+              <p className="font-normal text-xl pb-2">
+                {" "}
+                {
+                  "What technical areas of expertise does this project require? We love challenging ourselves and learning new skills, but we also want to ensure that our projects fit with our core capabilities."
+                }{" "}
+              </p>{" "}
+            </div>
+          </div>
+          <div className="flex max-lg:hidden w-5/12 justify-center items-center">
```

Comment:
Instead of hiding the right side content, we can adjust the breakpoints so that the right side content goes beneath the left side content as the screen gets smaller.